src/Application/Actions/Person/SetFirstPassword.php

```diff
@@ -23,6 +24,32 @@
  * the secret email verification token code {@see EmailVerificationToken}, and the UUID of the account.
  *
  * They will get both encoded in a link when they make a donation.
+ *
+ * @OA\Post(
+ *     path="/v1/people/setFirstPassword",
```

Comment:
We do have a todo in FE code at https://github.com/thebiggive/donate-frontend/blob/cf206cfd526f68cd6585d148c16c8382c8f590a8/src/app/identity.service.ts#L291 to change this path - I can still do but would really love to move on from the ID-47 ticket. Will need updating here, in routes file, and in FE.

Reply:
Yep I think we can leave this for post GMF, not a big issue to launch the feature with the current one.